### ISSUE TITLE
Reddit Tweets Prediction

### ISSUE THREAD

**Issue by aryantiwari10** [prathimacode-hub/ML-ProjectKart#539]
Define You:

- [x] LGM-SOC'21 Participant 
- [ ] DevIncept Participant
- [x] Contributor

In this project first I first performed a exploratory data analysis on the given dataset which includes of data cleaning , data manipulation, data preprocessing , data visualization and after that we used NLP for the stopwords removal and cleaning, after using NLP we did the model building using different machine learning classification and regression algorithms and then predicted the accuracy of every model . In the model prediction part we used different machine learning algorithms . In each algorithm I had included the accuracy score , training score , confusion matrix, classification report  . While in the EDA part I have included different plots for the different visualizations of our dataset .Some plots which we used for visualizing the dataset are Histogram , Barplot , Boxplot, Heatmap , Scatter plot , Pairplot , Jointplot etc.

 The Libraries used are:

PANDAS

NUMPY

MATPLOTLIB

SEABORN

SCIPY

SKLEARN

NLTK

TENSORFLOW

KERAS

**Comment by prathimacode-hub** [comment]
Issue assigned. @aryantiwari10
